test/log_subscriber_test.rb

```diff
@@ -10,7 +10,7 @@ class LogSubscribersController < ActionController::Base
 
     def with_page_cache
       cache_page('Super soaker', '/index.html')
-      render nothing: true
+      render body: nil
```

Comment:
This should be `head :ok` per the deprecation warning: DEPRECATION WARNING: `:nothing` option is deprecated and will be removed in Rails 5.1. Use `head` method to respond with empty response body